docs/en/guide/mount_at_boot.md

```diff
@@ -6,21 +6,31 @@ slug: /mount_juicefs_at_boot_time
 
 # Mount JuiceFS at Boot Time
 
-This is a guide about how to mount JuiceFS automatically at boot.
+After JuiceFS has been successfully mounted, follow this guide to setup auto-mount on boot.
 
 ## Linux
 
-Copy `juicefs` as `/sbin/mount.juicefs`, then edit `/etc/fstab` with following line:
+Using `--update-fstab` flag with the mount command should automatically set things up for you:
```

Comment:
This requires 1.1+

Reply:
Since 1.1 is not released, we should put this new feature at the end of this section.